schema.graphql

```diff
@@ -61,7 +61,7 @@ type Patron @entity {
   address: Bytes! # address
   lastUpdated: BigInt!
   previouslyOwnedTokens: [Wildcard!]
-  tokens: [Wildcard]!
+  tokens: [Wildcard!]!
```

Comment:
This is an unrelated change. But it simplifies the UI. Now tokens is an array of `wildcards`, not an array for `option(wildcards)`

Reply:
Cool makes sense.